samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -131,7 +133,20 @@ def __enter__(self):
                                                       self._debug_args,
                                                       self._debugger_path)
 
-        self._check_docker_connectivity()
+        container_manager = self._get_container_manager(self._docker_network, self._skip_pull_image)
```

Comment:
By making it local, I was  trying to simplify reading, because it would avoid `self._` on every access, but if you think that is a negligible enhancement I will assign to self here. What do you think?

Reply:
Understood. It would be more clear if you assigned to self here